persistence/account.go

```diff
@@ -23,7 +23,7 @@ func (p PostgresContext) GetAccountAmount(address []byte, height int64) (amount
 }
 
 func (p PostgresContext) getAccountAmountStr(address string, height int64) (amount string, err error) {
-	ctx, txn, err := p.DB.GetCtxAndTxn()
+	ctx, txn, err := p.GetCtxAndTx()
```

Comment:
I don't want to see txn anywhere in this code anymore. Let's keep it consistent

Examples:

```go
ctx, txn, err := p.GetCtxAndTx()
```

```go 
func (p *PostgresContext) GetChainsForActor(
	ctx context.Context,
	txn pgx.Tx,
```